### ISSUE TITLE
determine SaaS authorization process

### ISSUE THREAD

**Issue by afeld** [18F/tts-tech-portfolio#46]
Specifically, for Low-impact software-as-a-service (SaaS) past it's first year of approval, where systems are being asked to go from LISaaS to FedRAMP Tailored. [From Slack](https://gsa-tts.slack.com/archives/C039MHHF8/p1563377324122400), re: putting things through [FedRAMP Tailored](https://tailored.fedramp.gov/) for GSA use:

> The cost for sponsoring is "a one time fee of $42,000 for the initial authorization support, and $25,000 per year thereafter for Continuous Monitoring support" which would be paid for by the requesting office at GSA.

Need to get in touch with GSA IT and ensure that we find a solution that is financially reasonable, particularly for software whose licenses cost far less than that number.

- [x] Send letter
- [x] Meet with Security
- [x] Meet with FedRAMP
- [ ] Updated internal-facing documentation around the current steps/conditions
- [ ] Updated external (vendor)-facing documentation

**Comment by brodygov** [comment]
In May, Bo (GSA CISO) quoted these costs to Login.gov for sponsoring a FedRAMP authorizations:

Impact | GSA A&A Costs — one time | GSA CM Costs — annual
-- | -- | --
Low | $42,000 | $25,050
Moderate | $164,025 | $37,650

We've pushed back on this on a service-by-service individual basis and won 1-year reprieves for several of these. There are many services authorized in GEAR with LiSaaS ATOs that were renewed after the policy to require FedRAMP was imposed, so there is obviously some discretion here. The original directive that established FedRAMP also does provide a mechanism for agency CIOs to accept certain services that cannot be FedRAMPed and to provide a list of these to OMB annually.

**Comment by afeld** [comment]
- [IT Standards 2.0](https://docs.google.com/presentation/d/1FqIAPYZkYRsopCb3mDpenrIe2Pi2bWLkOTj9U8pLb1M/edit)
- [Improving cloud-based services use in TTS](https://docs.google.com/document/d/1_N1YkUGzBMIwJEv53VQUpfYVxykjYHvWXeOkrpFER4I/edit?ts=5d485306)

**Comment by afeld** [comment]
Sent the latter to the CIO, CTO, CISO, and some TTS leadership.

**Comment by annepetersen** [comment]
I see this was sent 12 days ago. Do we have an answer here? We have pending renewals that are becoming _quite_ urgent that are being held up by this.

Bottom line: _We seriously risk losing access to software we depend on if we don't resolve this soon._

**Comment by tbaxter-18f** [comment]
@afeld is there any update? We're at ~3 weeks since any news, which is a lifetime when the software you need to do your job is about to go kablooey. 

And if there isn't an update, what is our escalation point?

**Comment by afeld** [comment]
Broke out a number of follow-up "move the needle" type tasks—linked above. They need to be made more actionable, but are a starting place. Narrowed the scope of this issue to be around understanding the current state/process.

**Comment by afeld** [comment]
On second thought, moved the documentation tasks to a new issue: https://github.com/18F/handbook/issues/1662
